MultimodalQnA/ui/gradio/utils.py

```diff
@@ -180,3 +180,10 @@ def get_b64_frame_from_timestamp(video_path, timestamp_in_ms, maintain_aspect_ra
         b64_img_str = convert_img_to_base64(frame)
         return b64_img_str
     return None
+
+
+def convert_audio_to_base64(audio_path):
+    "Convert .wav file to base64 string"
```

Comment:
Is this supposed to be `"""`?

Reply:
Thanks, I fix it here and in the function I copied it from